pkg/ceph/client/pool.go

```diff
@@ -127,15 +132,22 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 		}
 	}
 
+	// ensure that the newly created pool gets an application tag
+	args = []string{"osd", "pool", "application", "enable", newPool.Name, appName}
+	buf, err = ExecuteCephCommand(context, clusterName, args)
+	if err != nil {
+		return "", fmt.Errorf("failed to enable application %s on pool %s. %+v", appName, newPool.Name, err)
+	}
+
 	logger.Infof("creating pool %s succeeded, buf: %s", newPool.Name, string(buf))
-	return string(buf), nil
+	return fmt.Sprintf("pool '%s' created", newPool.Name), nil
```

Comment:
`rookctl` simply prints the message returned from API calls (e.g. create block in addition to this call for create pool).  this was an attempt to maintain parity in the experience.  perhaps what we should be doing upon success is to return the message from the create pool call, not from the enable application call.  i think updating the client to construct its own messages would be out of scope for this issue.